sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -551,6 +576,49 @@ export class OfferingValidationModel {
     ),
   })
   yearOfStudy: number;
+  /**
+   * Indicates if the offering is an aviation offering.
+   */
+  @IsEnum(AviationYesNoOptions, {
+    message: getEnumFormatMessage(
+      userFriendlyNames.isAviationOffering,
+      AviationYesNoOptions,
+    ),
+  })
+  @ValidateIf((offering: OfferingValidationModel) => !!offering.programContext)
+  @ProgramIsAviationProgram(userFriendlyNames.isAviationOffering, {
+    context: ValidationContext.CreateWarning(
+      OfferingValidationWarnings.ProgramNotAviation,
+    ),
+  })
+  isAviationOffering: AviationYesNoOptions;
+  /**
+   * Indicates the aviation credential for the offering.
+   */
+  @IsEnum(AviationCredentialTypeOptions, {
+    message: getEnumFormatMessage(
+      userFriendlyNames.aviationCredentialType,
+      AviationCredentialTypeOptions,
+    ),
+  })
+  @ValidateIf(
+    (offering: OfferingValidationModel) =>
+      offering.isAviationOffering === AviationYesNoOptions.Yes,
+  )
+  @IsIn(aviationCredentialTypesEligibleForFunding, {
+    context: ValidationContext.CreateWarning(
+      OfferingValidationWarnings.AviationCredIsPrivatePilotTraining,
+    ),
+  })
+  @MatchProgramAviationCredential(
+    userFriendlyNames.matchProgramAviationCredential,
+    {
+      context: ValidationContext.CreateWarning(
+        OfferingValidationWarnings.MatchProgramAviationCredential,
+      ),
+    },
+  )
+  aviationCredentialType?: AviationCredentialTypeOptions;
```

Comment:
`@AllowIf` can be used to ensure this value is not present when it is not an aviation offering.